### PR TITLE
#167727750 manager re-confirm travel request

### PR DESCRIPTION
**Pull request by dinorhythms**
#### What does this PR do?
This PR allows a manager to be able to re-confirm travel requests
#### Description of Task to be completed?
- create re-confirm middleware
- add middleware to accept and reject travel request endpoints  
- test api endpoint   
#### How should this be manually tested?
- clone the repo
- Install dependencies, setup database and run npm start:dev
- make a post request to `/api/v1/requests/accept/:requestId` or `/api/v1/requests/reject/:requestId`
#### What are the relevant pivotal tracker stories?
[#167727750](https://www.pivotaltracker.com/story/show/167727750)

### CODE REVIEW

File: src/middlewares/confirmationMiddleware.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * @param {Object} req - Express request object
+ * @param {Oject} res - Express response object
+ * @param {Function} next - Express next middleware function
+ * @returns {Object} - Returns Object
+ */
+const requestConfirmation = async (req, res, next) => {
+  const { requestId } = req.params;
+  const { confirmation } = req.query;
+  try {
+    if (confirmation === undefined || confirmation === 'false') {
```
Comment (tejiri4) [inline src/middlewares/confirmationMiddleware.js]: LET USE JOI HERE

Comment (dinorhythms) [reply src/middlewares/confirmationMiddleware.js]: I don't think Joi is needed on it. I did a little update on it, please review

File: src/middlewares/confirmationMiddleware.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if (!confirmation) {
+      const option = {
+        include: [{
+          model: User,
+          as: 'User',
+          attributes: ['firstName', 'lastName']
+        }],
+        attributes: { exclude: ['id', 'approvalStatus', 'userId', 'createdAt', 'updatedAt', 'multiCity'] }
+      };
+      const request = await Request.findByPk(requestId, option);
```
Comment (tejiri4) [inline src/middlewares/confirmationMiddleware.js:28]: is error handled if request is not found?

Comment (dinorhythms) [reply src/middlewares/confirmationMiddleware.js:28]: Yes, it's handled. There are middlewares in the endpoints before this, that checks requestId. It won't get to this middleware if the request doesn't exist.

Comment (tejiri4) [reply src/middlewares/confirmationMiddleware.js:28]: Okay

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**